### ISSUE TITLE
environment target URL is never used

### ISSUE THREAD

**Issue by antony** [shoreditch-ops/artillery#64]
As discussed on gitter, environment specific target urls are not picked up.

For example:
```
"config": {
  "target": "xyz"
  "environments": {
    "perf": {
      "target": "http://my-api.example.net"
    }
  }
}
```

Then running with '-e perf' results in all requests actually being prefixed with: ```xyz``` rather than the environment's URL.

Leaving the config.target element out entirely results in the url being prefixed with ```undefined```

Expected behaviour is that the URL /v1/user would become ```http://my-api.example.net/v1/user``` rather than ```xyz/v1/user```.

**Comment by hassy** [comment]
Fix incoming.

**Comment by hassy** [comment]
@antony published `v1.3.9` with a fix (+ the fixes you contributed)
